client/scripts/app.ts

```diff
@@ -620,7 +620,7 @@ $(() => {
     '/:scope/analytics':         importRoute('views/pages/stats', { scoped: true, deferChain: true }),
     '/:scope/web3login':         importRoute('views/pages/web3login', { scoped: true }),
 
-    '/:scope/account/:address':  importRoute('views/pages/profile', { scoped: true, deferChain: true }),
+    '/:scope/account/:address':  importRoute('views/pages/profile', { scoped: true, deferChain: false }),
```

Comment:
Any reason why we now require loading the chain to view an address? This will slow down loading times.

Reply:
I don't remember this correctly. But in my remember, I did a discussion with @raykyri to change this to load the chain for claiming address.